src/scripts/main.js

```diff
@@ -1,11 +1,26 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    const eventHandler = (e) => {
+      element.removeEveventListener(eventName, eventHandler);
+
+      resolve(`It was ${eventName} on the element:
+        ${element.nodeName}, id: ${element.id}.`);
```

Comment:
There is a syntax error in the template literal. The string should be enclosed properly with backticks (`) and the template literal should be correctly formatted. Consider using a single line or ensuring the multiline string is properly formatted.